lib/colors.ts

```
@@ -0,0 +1,114 @@
+  rgb: string
+  usage?: string
+}
+
+/**
+ * Convert hex color to RGB format
+ */
+const hexToRgb = (hex: string): string => {
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
+  if (!result) return hex
+
+  const r = parseInt(result[1], 16)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)